extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -46,7 +46,7 @@ export class AmpIframe extends AMP.BaseElement {
         this.element);
     const containerUrl = parseUrl(containerSrc);
     assert(
-        !((' ' + sandbox + ' ').match(/\s+allow-same-origin\s+/)) ||
+        !((' ' + this.sandbox_ + ' ').match(/\s+allow-same-origin\s+/)) ||
```

Comment:
This regex test is weird, and it won't allow insensitive matches. I think it'd be better to [normalize `this.sandbox_` into a string](https://github.com/sriramkrish85/amphtml/blob/f95f920/extensions/amp-iframe/0.1/amp-iframe.js#L222), then use `!(/\s*allow-same-origin\s*/i).test(this.sandbox_)`.